node/core/chain-selection/src/tree.rs

```diff
@@ -561,3 +562,102 @@ pub(super) fn detect_stagnant<'a, B: 'a + Backend>(
 
 	Ok(backend)
 }
+
+/// Revert the tree to the block relative to `hash`.
```

Comment:
What does "relative to `hash`" mean? `hash` doesn't convey a semantic ordering and no block offset number is provided.

Reply:
The function expects the caller to pass the `hash` of the revert point, computed [here](https://github.com/paritytech/polkadot/blob/43f6e9087222280618f63d7f77a882f4e272ef59/node/service/src/lib.rs#L1403-L1413).

The main motivation is that this method is called along with babe/grandpa revert procedures. 
In these procedures we revert from the "best" block according to the `client.info().best_number` (i.e. `ChainInfo::best_number`).

Given that this chain selection has its internal machinery to determine what is the best number, I'm not sure that here (if we pass only the offset) I can get to the same revert point as computed by grandpa/client (without passing directly the best number according to the `ClientInfo`).

But if is guaranteed that the first entry of the leaves returned by [`load_leaves`](https://github.com/paritytech/polkadot/blob/43f6e9087222280618f63d7f77a882f4e272ef59/node/core/chain-selection/src/backend.rs#L45) is equal to the `client.info().best_number` then I can perform the revert point computation directly in the revert procedure.

What you suggest?

Reply:
Renaming the method to `revert_to` may clarify the semantic of `hash` param.